routes/major.js

```diff
@@ -1,5 +1,7 @@
 const majors = require("../model/Major.js");
 const { returnData, errorHandler } = require("./helperMethods.js");
```

Comment:
thoughts on replacing all instances of helperMethods.js by helperMethods.ts? 

helperMethods.ts has returnData and errorHandler too

Reply:
separated returnData and errorHandler from distribution functions